addons/web/static/src/core/datepicker/datepicker.js

```diff
@@ -277,7 +283,7 @@ DatePicker.defaultProps = {
 DatePicker.props = {
     // Components props
     onDateTimeChanged: Function,
-    date: DateTime,
+    date: { validate: (dt) => dt instanceof DateTime || !dt, optional: true },
```

Comment:
not sure, but I think we can now use `date: { type: [Datetime, {value: false}, optional: true}`